Fabulous.XamarinForms/samples/FabulousWeather/FabulousWeather/PancakeViewExtensions.fs

```diff
@@ -0,0 +1,77 @@
+﻿namespace FabulousWeather
+
+open Fabulous
+open Fabulous.XamarinForms
+
+[<AutoOpen>]
+module PancakeViewExtensions =
+
+    // Define keys for the possible attributes
+    let backgroundGradientStartColorAttribKey = AttributeKey "backgroundGradientStartColorAttribKey"
```

Comment:
The general rule of AttributeKey is to have their string value PascalCase without the AttribKey suffix (e.g. `BackgroundGradientStartColor`)